src/Infrastructure/Identity/UserService.CreateUpdate.cs

```diff
@@ -120,7 +130,10 @@ public async Task<string> CreateAsync(CreateUserRequest request, string origin)
             throw new InternalServerException(_localizer["Validation Errors Occurred."], result.GetErrors(_localizer));
         }
 
-        await _userManager.AddToRoleAsync(user, FSHRoles.Basic);
+        if (!isAnonymous)
```

Comment:
I don't think this check is necessary... the "basic" role is supposed to be just that... very basic... so for new users.

Reply:
Even if you would do the check, I wouldn't create a separate method for it on the interface (SelfRegisterAsync)... might then as well just take the bool as an extra argument in the CreateAsync call.

Reply:
ok, will adjust the code in this PR just use the current API for `CreateAsync` to keep things simple

Reply:
code adjusted @iammukeshm  @fretje

Reply:
@274188A Euhm... all the code seems to be gone again?

Reply:
good point - i'll just close this PR as not required/does nothing..